### PR TITLE
delete button activated

### PR DESCRIPTION
**Pull request by alecorra**

### CODE REVIEW

File: deleteForm.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php require 'editFetchComand.php' ?>
```
Comment (Liliya-V) [inline deleteForm.php]: Why is it called deleteForm, Is't is an admin panel itself?

Comment (alecorra) [reply deleteForm.php]: fixed

File: deleteComand.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+$db = new PDO('mysql:host=192.168.20.20;dbname=portfolioProject', 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql = "UPDATE `portfolio` SET `on_to` = 0 WHERE `id` = :id;";
+
+$sqlCheck = "SELECT `id` FROM `portfolio`;";
+
+$checkQuery = $db->query($sqlCheck);
+$checkResult = $checkQuery->fetchAll();
```
Comment (mporam) [inline deleteComand.php]: why are you running another select? you dont need to, if the id doesnt exist it will just do nothing which is fine

Comment (alecorra) [reply deleteComand.php]: fixed

**Pull request merged**